modules/hal_gigadevice/Kconfig

```diff
@@ -23,7 +23,7 @@ if HAS_GD32_HAL
 choice GD32_HXTAL_FREQUENCY
 	prompt "High speed external oscillator clock frequency"
 	default GD32_HXTAL_FIRMWARE_DEFINED if !SOC_SERIES_GD32VF103
```

Comment:
May have conflict with below new add `|| SOC_SERIES_GD32E50X`?

Reply:
right, will fix it!

Reply:
done